### ISSUE TITLE
Grouped cards in search are not relevant

### ISSUE THREAD

**Issue by akshaybhalotia** [hasgeek/hasjob#421]
Job cards from the same company are grouped under one, and user has to click to show all. But this behavior is not expected in search, where only the card with the relevant job should show up. Eg. If a company has openings for both iOS and Android developers with the iOS opening being posted after Android, a search for keyword Android would show up the card stack with iOS job as the front card.

**Comment by jace** [comment]
This should only happen if the iOS job also mentions Android. Do you have an example?

**Comment by akshaybhalotia** [comment]
Just a mention of the word in description under `What We're Building` triggers this: check https://hasjob.co/vebbler.com/xrt2g for example. It appears on Android keyword search currently.
![screen shot 2018-03-05 at 3 02 43 pm](https://user-images.githubusercontent.com/9196911/36967735-92d26d06-2086-11e8-813f-d506b015d04a.png)

**Comment by jace** [comment]
Our keyword search is based on full-text search, so this is expected behaviour. However:

1. There is a separate concern with the relevancy of keywords in the job description. We do not have the ability to extract only relevant keywords from the text (whether in an automated or manual fashion). This merits its own ticket.

2. There is a valid argument for saying _keyword_ search results should not be grouped. If we accept this, we can remove grouping when keywords are used.

**Comment by akshaybhalotia** [comment]
Yes, point 2 is what I meant. Will file a separate one for relevancy matching.
